dclab/rtdc_dataset/feat_anc_plugin/plugin_feature.py

```diff
@@ -32,6 +32,10 @@ def import_plugin_feature_script(plugin_path):
     ------
     PluginImportError
         If the plugin can not be found
+
+    Notes
+    -----
+    There can be several plugin features being calculated in one recipe.
```

Comment:
Please write this in a less passive voice, more plain and simple: "One recipe may define multiple plugin features." (also below in `load_plugin_feature`

Reply:
Alright!